### PR TITLE
Feature/go hello

### PR DESCRIPTION
**Pull request by d01a**
test file for go binary identification 
> note: the makefile commands are for linux. in windows command will be slightly different

### CODE REVIEW

**Comment by d01a** [comment]
there was a file identified as malware so it will be missed in my PR. AV deleted it sry 😄

**Comment by d01a** [comment]
I deleted the make instructions and restored the malware. 
How can i add checks?

File: src/Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ TESTS= \
 	decode-reencode-string \
 	decode-wrapped-decoder \
 	decode-to-stack-rep-mov \
-    decode-substitution-cipher
+    decode-substitution-cipher
```
Comment (mr-tz) [inline src/Makefile:21]:
```suggestion
    decode-substitution-cipher
```

**Comment by mr-tz** [comment]
note, that `tests/data` needs to be updated manually in the `flare-floss` repo to point to the current commit

**Pull request merged**